cores/rp2040/CoreMutex.cpp

```diff
@@ -23,19 +23,20 @@
 
 #include "Arduino.h"
 #include "CoreMutex.h"
+#include "projdefs.h"
```

Comment:
Think we got a stray include here...

Reply:
I got a compile error on pdFALSE. Tried to find an include that holds the definition.

PS : Same for `BaseType_t` 

Will check.

PS 2 : Will be out for my break in 2 hours time